korebuild-lock.txt

```diff
@@ -1,2 +1,2 @@
-version:2.1.0-preview1-15679
-commithash:5347461137cb45a77ddcc0b55b2478092de43338

```

Comment:
@pranavkm @natemcmaster is this right? What's the right way to update Korebuild/sdk?

Reply:
Note: this is the release branch

Reply:
I know @pranavkm signed off on this bad boy but can I get one of you to confirm that this is right for the release/2.1 branch? @natemcmaster @pranavkm  plz.

Reply:
Normally the right way to update korebuild is `run.ps1 -update`. In this case, korebuild.json is misconfigured to point to the `dev` branch of build tools. Update it to say `"channel": "release/2.1"` and try `run.ps1 -update` again.

Reply:
And no, build tools 2.1.0-preview2-15688 is not the right one to use. The latest release/2.1 build tools is 2.1.0-preview1-1005

See https://github.com/aspnet/BuildTools#latest-build

Reply:
So does this need to be updated in all of our release branches then?

Reply:
Yeah. I was planning to do it. Just waiting to get a build of the SDK that actually works. (blocked on https://github.com/aspnet/BuildTools/issues/562)

Reply:
OK cool, I logged an issue in aspnet/Universe so it didn't get buried. Looks like you already found that though